core/platform/datastore/gae_datastore_services.py

```diff
@@ -31,7 +31,18 @@
 from google.appengine.datastore import datastore_stub_util
 from google.appengine.ext import ndb
 
+from typing import ( # isort:skip # pylint: disable=unused-import
+    Any, Callable, Iterator, List, Optional, Sequence,
+    Text, Tuple, TypeVar)
+
+MYPY = False
+if MYPY:
+    from mypy_imports import ( # pragma: no cover # pylint: disable=unused-import
+        base_models, transaction_services) # pragma: no cover
```

Comment:
Do you actually need "no cover" on this line given that it's already on the line above and this is just a continuation?

Reply:
I think adding this pragma to if MYPY: only will fix this. Did this